src/UI/Text.elm

```diff
@@ -14,251 +15,100 @@ module UI.Text exposing
     , subtitle1
     , subtitle2
     , toEl
+    , withColor
     )
 
 import Element exposing (Attribute, Element)
 import Element.Font as Font
 import List
+import UI.Internal.Text as Internal exposing (..)
+import UI.Palette as Palette
```

Comment:
Should we move this module to `UI.Internals.Palette` as well? this way it's more explicit that this module is not "exposable/exposed"

Reply:
There is a `UI.Internals.Pallete` and a `UI.Palette`